src/wrapped_radix_engine_toolkit/high_level/token_definition_metadata.rs

```diff
@@ -41,7 +56,7 @@ impl From<TokenDefinitionMetadata>
                 ScryptoMetadataValue::StringArray(value.tags),
             ),
             (
-                "extra_String".to_string(),
+                "extra_string".to_string(),
```

Comment:
Any reason why those keys are string literals instead of variants on the MetadataKey (or at least another new enum for ExtraMetadataKeys)?

Reply:
I've named them `extra_XYZ` because they don't belong to the "predefined" metadata keys that we search for its values in the host apps (`name`, `symbol`, `icon_url`). They are named however the resource creator decided to name them.

We won't be searching for `extra_string` on the host app, just displaying the key/value pair. In this case it is `"extra_string": "foo_bar"`, but it could have been anything else (`"whatever": "lorem ipsum"`).